### ISSUE TITLE
softmax of the attention vector

### ISSUE THREAD

**Issue by batman47steam** [suvojit-0x55aa/A2S2K-ResNet#4]
`attention_vector = torch.cat(
            [
                self.conv_ex(Z).unsqueeze(dim=1),
                self.conv_ex(Z).unsqueeze(dim=1)
            ],
            dim=1)`
        `attention_vector = self.softmax(attention_vector)`
and `self.softmax = nn.Softmax(dim=1)`
it seems that the elements of the attention_vector are the same, so if you apply softmax on `dim=1`，the result of the softmax will all be the same, 0.5 for sure

so why are we doing this，i don't know if i have missed something

<img width="177" alt="image" src="https://user-images.githubusercontent.com/36287035/161505111-5092b938-313c-486e-bb3b-b7afefb9ca03.png">
